shoulder/model/access_mechanism/mcr.py

```diff
@@ -42,9 +42,6 @@ class MCR(AbstractAccessMechanism):
     crm: bytes
     """ Operational register within CRn """
 
-    operand_mnemonic: str
-    """ The operand mnemonic of the register to be accessed """
-
```

Comment:
I noticed that all these coprocessor mechanisms removed this `operand_mnemonic`. Do coprocessor registers not have a mnemonic? Or is it just not useful?

Reply:
Yea, after a high-level look through how MRC/MRRC/MCR/MCRR works in assembly, they don't have any operand mnemonics. To access coprocessors in assembly, you actually end up writing something out that resembles the instruction's encoding like this:

```
mrc p15, #1, x0, c1, c2, #3
```

Reply:
Ok cool, that's what I figured. I just wanted to make sure